src/chapters/unit-testing/tdd-example.rst

```diff
@@ -0,0 +1,560 @@
+TDD in Action
+=============
+
+Fork `this repl.it <https://repl.it/@launchcode/Transmission-processor-TDD-starter>`_
+and follow along as we implement a project using TDD.
+
+We need to write a Node module to process transmissions from the
+`Voyager1 probe <https://voyager.jpl.nasa.gov/mission/>`_.
+
+.. admonition:: Example
+
+   Transmission
+
+   .. sourcecode:: js
+
+      "1410::<932829840830053761>"
+
+   Expected Result
+
+   .. sourcecode:: js
+
+      {
+         id: 1410,
+         rawData: 932829840830053761
+      }
+
+
+Requirements
+------------
+The features for this project have already been broken down into
+small testable units. Let's review these and then we will
+take it slow and handle these one at a time.
+
+#. Take in a transmission string and return an object ``{}``.
```

Comment:
Using `{}` could be interpreted as an empty object literal, which is not what I think you're saying here.

Reply:
The end goal is an object that has properties, but I wanted to start by having the students return an empty object. @jimflores5 how does this feel to you? I'm open to changing it if the step feels wrong.

Reply:
I'm OK with starting with an empty object, though I would prefer that to be explicitly stated.

"Take in a transmission string and return an empty object ``{}``."
OR
""Take in a transmission string and return an object with no assigned key/value pairs ``{}``."

Reply:
So the test doesn't check if the object returned is empty, it simple checks if the type is object. Are you ok with leaving it as is?

Reply:
OK.